tahoe_sites/__init__.py

```diff
@@ -1,5 +1,5 @@
 """
 tahoe-sites app initialization module
 """
-__version__ = '1.2.0'  # pragma: no cover
+__version__ = '1.3.0'  # pragma: no cover
```

Comment:
If the version is being bumped, the CHANGELOG also needs updating.

Reply:
Thanks, Done!